src_py/sysfont.py

```diff
@@ -54,10 +55,13 @@ def _addfont(name, bold, italic, font, fontdict):
 
 def initsysfonts_win32():
     """initialize fonts dictionary on Windows"""
-
-    fontdir = join(os.environ.get("WINDIR", "C:\\Windows"), "Fonts")
-    fonts = {}
-
+	#faster and quicker
+	try:
+		fontdir = join(os.getenv("WINDIR", "C:\\Windows"), "Fonts")
+		fonts = {}
+	except ImportError as e:
+		print(e)
+			
```

Comment:
This change is incorrect. Wrapping a call to `getenv()` in a try except block does not make this code faster or quicker as the comment implies.

`getenv()` just returns None if it the environment variable cannot be found, it doesn't raise an `ImportError`. I suggest reading the python documentation for the os module:

https://docs.python.org/3/library/os.html

Reply:
got it !